INSTALL.md

```diff
@@ -41,13 +44,22 @@ sudo pacman -S mono openal libgl freetype2 sdl2 lua51 xdg-utils zenity
 Debian/Ubuntu
 -------------
 
+:warning: The `mono` packages in the Ubuntu < 19.10 and Debian < 10 repositories too old to support OpenRA. :warning:
```

Comment:
"are" missing in that sentence. And don't you mean Ubuntu < 19.04?  Disco will include mono 5.16.

Reply:
Good catch. I *meant* 19.04, no idea why I wrote 19.10.